docs/sdk-and-tools/notifier.md

```diff
@@ -48,13 +48,70 @@ In the figure above:
 
 ### Observer Client
 
-On the observer side, there is a http client that will push block events to notifier service.
+On the observer side, there is a WebSocket client that will push block events to notifier service.
+There is also the HTTP Integration, which will be deprecated in the future.
 
-In the observer node's configuration directory, `external.toml` config file can be configured
-to enable notifier connector. The config file can be found 
+In the observer node's configuration directory, the `external.toml` config file can be configured
+to enable events notifier connector via WebSocket or via HTTP integrations. The config file can be found 
 [here](https://github.com/multiversx/mx-chain-go/blob/master/cmd/node/config/external.toml).
 
-The supported config variables are as follows:
+[comment]: # (mx-context-auto)
+
+#### WebSocket Integration
```

Comment:
4 `#` won't be included in the right side's "table on content". Can remain as it is, tho